modules/develop/partials/run-transforms.adoc

```diff
@@ -333,10 +440,11 @@ See xref:reference:public-metrics-reference.adoc[]
 - Only a single output topic is supported.
 - Transforms have at-least-once delivery.
 - When clients use the Kafka Transactions API on partitions of an input topic, transforms process only committed records.
-- Because data transforms are powered by Wasm, transform functions can be authored in any language. However, a data transforms SDK currently is only available in xref:reference:data-transform-api.adoc[Golang].
+- Because data transforms are powered by Wasm, transform functions can be authored in any language. However, a data transforms SDK currently is only available in xref:reference:data-transform-api.adoc[Golang] and xref:reference:data-transform-rust-sdk.adoc[Rust].
 
 == Suggested reading
 
 - xref:./how-transforms-work.adoc[]
 - xref:reference:data-transform-api.adoc[]
```

Comment:
Now that we are going to have these per language, should this be `data-transform-golang-sdk`?